src/DocumentFeedMigrator.cs

```diff
@@ -4,22 +4,27 @@
 
 namespace MigrationExecutorFunctionApp
 {
+    using Microsoft.Azure.Cosmos;
+    using Microsoft.Azure.Documents;
+    using Microsoft.Azure.WebJobs;
+    using Microsoft.Extensions.Logging;
     using System;
     using System.Collections.Concurrent;
     using System.Collections.Generic;
     using System.Diagnostics;
     using System.Linq;
+    using System.Text;
     using System.Threading;
     using System.Threading.Tasks;
-    using Microsoft.Azure.Cosmos;
-    using Microsoft.Azure.Documents;
-    using Microsoft.Azure.WebJobs;
-    using Microsoft.Extensions.Logging;
+
+
 
     public class DocumentFeedMigrator
     {
         private Container containerToStoreDocuments;
-
+        string targetPartitionKeyAttribute = Environment.GetEnvironmentVariable($"{"TargetPartitionKeyAttribute"}");
+        static string sourcePartitionKeyMapping = Environment.GetEnvironmentVariable($"{"SourcePartitionKeyMapping"}");    
+        Boolean isSyntheticKey = Environment.GetEnvironmentVariable($"{"SourcePartitionKeyMapping"}").IndexOf(",") == -1? false: true;
```

Comment:
Shouldn't be checking the TargetPartitionKeyAttribute?

Reply:
I only need to check whether there are many attributes, or one, in the source partition key field. For TargetPartitionKeyAttribute I am assuming they provide a valid single field. Perhaps we could add a check. But these fields need documenting in the readme, which I have not done yet.